torch/csrc/Module.cpp

```diff
@@ -493,7 +496,7 @@ PyObject *THPModule_inferSize(PyObject *_unused, PyObject *args)
   THLongStorage *sizes = sizes_guard.get();
 
   char error_buffer[1024];
-  int ret = THLongStorage_inferSize2(sizes, size1->data, size1->size, size2->data, size2->size, error_buffer, 1024);
+  int ret = (int) THLongStorage_inferSize2(sizes, size1->data, size1->size, size2->data, size2->size, error_buffer, 1024);
```

Comment:
why is this cast even necessary? What does it return normally? Maybe just change the type of `ret`?

Reply:
I'll remove this useless cast.